bee-metrics/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "bee-metrics"
+version = "0.1.0"
+authors = [ "IOTA Stiftung" ]
+edition = "2021"
+description = "Track metrics using prometheus"
```

Comment:
```suggestion
description = "Track metrics using prometheus."
```

Reply:
If I remember correctly, we don't do the `.` at the end of descriptions, also just checked it with our other crates

Reply:
Seems like we're inconsistent. In some crates we use punctuation. I checked one of the reference projects - `tokio` aaaand they are inconsistent as well :joy: , but mostly use punctuation I think (didn't check all their sub-crates so might be wrong).
I am in strong favor of using punctuation since the capital letter at the beginning indicates a PROPER sentence. Why do I hear the bike-shedding crowd screaming in the back of my head now????

Reply:
Haha if even tokio isn't consistent, I guess lots of other Rust projects don't know how to handle it best too 🤣
We should be consistent at least imo. In the Rust docs it is without point https://doc.rust-lang.org/cargo/reference/manifest.html#the-description-field

If I remember we had a brief discussion about it months ago to use punctation only for multi-line descriptions.

Reply:
I can't remember anymore. Maybe you're right. I don't think we should make punctuation dependent on whether it's a multi-line description or not. Also there's really not much effort to put it and then it becomes a wonderful syntactically correct sentence which I have a bit of a faible for. :heart:

Reply:
I fully agree that it looks nicer and it was also my preferable choice. I actually just think we should be consistent (similarly like with the error messages). @thibault-martinez what is your suggestion here?

Reply:
Feel free to propose it later so we change it repository-wise but I wouldn't introduce an inconsistency in this PR.